### PR TITLE
aligned progress bar and create directory if not exists 

### PR DESCRIPTION
**Pull request by unique1o1**

### CODE REVIEW

**Comment by unique1o1** [comment]
clean up

File: gitir_downloader/main.py [hunk 1]
```diff
@@ -1,10 +1,9 @@
+import argparse
 import os
 import sys
-import argparse
 
-from .parser import parse_url
 from .downloader import download_files
-
+from .parser import parse_url
```
Comment (yankeexe) [inline gitir_downloader/main.py:6]: Bharyang imports. Smaller line lengths on top of longer ones. 

```py
from .parser import parse_url
from .downloader import download_files
```

File: gitir_downloader/main.py [hunk 2]
```diff
@@ -32,10 +31,10 @@ def init_argparse():
 
 def start():
     """
-    entrypoint for the app
+    entry-point for the app
     """
     try:
-        args = init_argparse()
+        args: argparse.Namespace = init_argparse()
```
Comment (yankeexe) [inline gitir_downloader/main.py:37]: Let's keep the type hinting on global variables and function arguments only.

File: gitir_downloader/parser.py [hunk 3]
```diff
@@ -1,15 +1,15 @@
 """ Parse URL and scrape relevant links. """
-import re
 import argparse
-from typing import List
+import re
+from typing import List, Tuple
```
Comment (yankeexe) [inline gitir_downloader/parser.py:4]: bharyang imports.

File: gitir_downloader/downloader.py [hunk 4]
```diff
@@ -24,8 +25,10 @@ def download_files(folder_title, LINKS, args: argparse.Namespace):
     # Create folder
     spinner = Halo(text="Creating folder", spinner="dots")
     spinner.start()
-
-    dir_path.mkdir()
+    try:
+        dir_path.mkdir()
+    except FileExistsError as _:
+        pass
```
Comment (yankeexe) [inline gitir_downloader/downloader.py:31]: Display a custom message maybe? instead of pass?

**Pull request merged**